splunk-apm/README.md

```diff
@@ -59,21 +70,31 @@ No code change is needed to instrument a lambda. However, the `AWS_LAMBDA_EXEC_W
 - `/opt/otel-proxy-handler` for wrapping regular handlers implementing `RequestHandler`, proxied through API Gateway, and enabling HTTP context propagation
 - `/opt/otel-stream-handler` for wrapping streaming handlers implementing `RequestStreamHandler`
 
-#### Configuration example
+### Metrics extension
+
+The Splunk Extension for AWS Lambda provides customers with a simplified runtime-independent interface to collect high-resolution, low-latency metrics on AWS Lambda function execution. The Splunk Extension layer tracks metrics for cold start, invocation count, function lifetime, and termination condition. This enables customers to effectively monitor their AWS Lambda Functions with minimal overhead.
+
+Full documentation with examples can be found [here](https://github.com/signalfx/splunk-extension-wrapper/tree/main/docs).
+
+
+
+### Configuration example
 
 The example assumes the following:
 
+- Both SFx metrics and OTeL traces.
```

Comment:
```suggestion
- Both Splunk metrics and OTeL traces.
```

(Why SFx?)

Reply:
It's SFx tech - the metrics we're pushing here (as OTeL spec for metrics is not ready yet). I can call them "Splunk metrics" but I'm sure i will spark come confusion among current customers.

Reply:
I've updated my suggestions to remove product names/brands (unless it's necessary to make a distinction here). SFx is not really something we should be using anymore.